src/jit/codegenlegacy.cpp

```diff
@@ -13013,7 +13013,7 @@ void CodeGen::genCodeForBBlist()
 
             case BBJ_CALLFINALLY:
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && !FEATURE_EH_FUNCLETS
 
```

Comment:
Yes, but why is this file getting built with FEATURE_EH_FUNCLETS defined on x86?